com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UploadAppStep.java

```diff
@@ -81,17 +85,45 @@ public StepPhase executeAsyncStep(ExecutionWrapper execution) throws FileStorage
         return StepPhase.POLL;
     }
 
+    private ApplicationResources getApplicationResources(ExecutionWrapper execution, CloudControllerClient client, CloudApplication app,
+        String appArchiveId,
+        String fileName)
+        throws FileStorageException {
+        ApplicationResources applicationResources = new ApplicationResources();
+        final DelegateExecution context = execution.getContext();
+        FileDownloadProcessor calculateDigestOfApplication = new DefaultFileDownloadProcessor(StepsUtil.getSpaceId(context),

```

Comment:
Why the variable is called `calculateDigestOfApplication ` when it is refered to a FileDownloadProcessor?